extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -229,7 +235,7 @@ class AmpVideoIframe extends AMP.BaseElement {
   getSrc_() {
     const {element} = this;
     const urlService = Services.urlForDoc(element);
-    const src = urlService.assertHttpsUrl(element.getAttribute('src'), element);
```

Comment:
This assertion was redundant. User `createFrameFor()` already asserts for `https`.